ansible/inventories/production/group_vars/inventory-next/vars.yml

```diff
@@ -1,5 +1,5 @@
 ---
-datagov_service: inventory
+datagov_service: inventory-next
```

Comment:
👍  does this also need to change in other environments (staging)?

Reply:
Yes! Thank you. Just made this change to sandbox/staging inventories as well